airflow/models/__init__.py

```diff
@@ -3409,11 +3409,10 @@ def handle_callback(self, dagrun, success=True, reason=None, session=None):
             callback(context)
 
     @provide_session
```

Comment:
These decorators should also be removed.

Reply:
Tnx!